pkg/fleetautoscalers/controller.go

```diff
@@ -169,30 +192,46 @@ func (c *Controller) validationHandler(review admissionv1.AdmissionReview) (admi
 	return review, nil
 }
 
-// syncFleetAutoscaler scales the attached fleet and
-// synchronizes the FleetAutoscaler CRD
+// syncFleetAutoscaler syncs FleetAutoScale according to different sync type
 func (c *Controller) syncFleetAutoscaler(ctx context.Context, key string) error {
-	c.loggerForFleetAutoscalerKey(key).Debug("Synchronising")
+	if runtime.FeatureEnabled(runtime.FeatureCustomFasSyncInterval) {
+		return c.syncFleetAutoscalerWithCustomSyncInterval(ctx, key)
+	}
+	return c.fleetAutoScale(ctx, key)
+}
 
+// getFleetAutoscalerByKey get the Fleet Autoscaler by key
```

Comment:
```suggestion
// getFleetAutoscalerByKey get the Fleet Autoscaler by key
// a nil FleetAutoscaler returned indicates that an attempt to sync should not be retried, e.g.  if the FleetAutoscaler no longer exists.
```

Would suggest a small note here to explain what the convention here is if there is a nil FleetAutoscaler value.